pymeasure/adapters/prologix.py

```diff
@@ -89,6 +89,30 @@ class PrologixAdapter(VISAAdapter):
         sudo udevadm control --reload-rules
         sudo udevadm trigger
 
+    Since the Prologix adapter uses the same communication channel (an USB
+    CDC) for both, communication with the adapter itself, as well as
+    communication over GPIB, certain things need to be kept in mind:
+
+    - Operations that need to read from GPIB use the standard :read:
+      method.
+
+    - Operations that just read responses from the Prologix itself need to
+      add the parameter :prologix=True: to :read:; this avoids requesting
```

Comment:
This is sphinx style and will make it a link (to `read` method) in the generated documentation:
```suggestion
      add the parameter :code:`prologix=True` to :meth:`.read`; this avoids requesting
```

Reply:
OK, I found the Sphinx Makefile. I'll give that a try, and look at your suggested changes.